File: src/main/java/com/semiprj/data/RoomAccessInfo.java
package com.semiprj.data;


import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class RoomAccessInfo {
    public RoomAccessInfo(){}
    public RoomAccessInfo(Room room, CamClient rsProfile){
        this.room = room;
        this.rsProfile = rsProfile;
    }

    private Room room;
    private CamClient rsProfile;
}

File: src/main/webapp/resources/frame/data/RoomAccessInfo.js
export default class RoomAccessInfo{
    constructor (){
        this._room;
        this._rsProfile;
    }

    spawn(room, rsProfile){
        this._room = room;
        this._rsProfile = rsProfile;
        return this;
    }

    get room(){
        return this._room;
    }

    get rsProfile(){
        return this._rsProfile;
    }
}